### PR TITLE
Add config details

### PR DESCRIPTION
**Pull request by dianashk**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,9 +37,17 @@ The API ships with several convenience commands (runnable via `npm`):
 ## pelias-config
 The API recognizes the following properties under the top-level `api` key in your `pelias.json` config file:
 
-  * `accessLog`: (*optional*) The name of the format to use for access logs; may be any one of the
+|parameter|required|default|description|
+|---|---|---|---|
+|`host`|*yes*||specifies the url under which the http service is to run|
+|`textAnalyzer`|*yes*|*addressit*|can be either `libpostal` or `addressit` however will soon be deprecated and only `libpostal` will be supported going forward|
```
Comment (trescube) [inline README.md]: If there's a default then it's not required

File: README.md [hunk 2]
```diff
@@ -37,9 +37,17 @@ The API ships with several convenience commands (runnable via `npm`):
...
+|`legacyUrl`|*no*||the url to redirect to in case the user does not specify a version such as `v1`
+|`relativeScores`|*no*|true|if set to true, confidence scores will be normalized, realistically at this point setting this to false is not tested or desirable
+|`accessLog`|*no*||name of the format to use for access logs; may be any one of the
   [predefined values](https://github.com/expressjs/morgan#predefined-formats) in the `morgan` package. Defaults to
-  `"common"`; if set to `false`, or an otherwise falsy value, disables access-logging entirely.
+  `"common"`; if set to `false`, or an otherwise falsy value, disables access-logging entirely.|
+|`pipService`|*yes*||full url to the pip service to be used for coarse reverse queries. if missing, which is not recommended, the service will default to using nearby lookups instead of point-in-polygon.|
```
Comment (trescube) [inline README.md]: This is probably going to move to a `services` child blob.

File: README.md (hunk 1 above)
Comment (dianashk) [inline README.md]: true

File: README.md (hunk 2 above)
Comment (dianashk) [inline README.md]: yeah until then i'd rather just document it so it's not confusing any longer. i had it actually calling out the new way and then removed it. :)

Comment (trescube) [inline README.md]: Cool, good idea!

File: README.md [hunk 3]
```diff
@@ -37,9 +37,53 @@ The API ships with several convenience commands (runnable via `npm`):
...
-  * `accessLog`: (*optional*) The name of the format to use for access logs; may be any one of the
+|parameter|required|default|description|
+|---|---|---|---|
+|`host`|*yes*||specifies the url under which the http service is to run|
+|`textAnalyzer`|*no*|*addressit*|can be either `libpostal` or `addressit` however will soon be **deprecated** and only `libpostal` will be supported going forward|
+|`indexName`|*no*|*pelias*|name of the Elasticsearch index to be used when building queries|
+|`legacyUrl`|*no*||the url to redirect to in case the user does not specify a version such as `v1`
+|`relativeScores`|*no*|true|if set to true, confidence scores will be normalized, realistically at this point setting this to false is not tested or desirable
+|`accessLog`|*no*||name of the format to use for access logs; may be any one of the
   [predefined values](https://github.com/expressjs/morgan#predefined-formats) in the `morgan` package. Defaults to
```
Comment (orangejulius) [inline README.md]: this newline appears to have broken things :(

**Pull request merged**